code/game/objects/structures/crates_lockers/closets/secure/freezer.dm

```diff
@@ -16,11 +16,13 @@
 	name = "kitchen cabinet"
 	req_access = list(access_kitchen)
 
-	fill()
-		for(var/i = 0, i < 2, i++)
-			new /obj/item/reagent_containers/food/condiment/flour(src)
-		new /obj/item/reagent_containers/food/condiment/sugar(src)
-		new /obj/item/reagent_containers/food/condiment/shaker/spacespice(src)
+/obj/structure/closet/secure_closet/freezer/kitchen/fill()
+	for(var/i = 0, i < 2, i++)
+		new /obj/item/reagent_containers/food/condiment/flour(src)
+	new /obj/item/reagent_containers/food/condiment/sugar(src)
+	new /obj/item/reagent_containers/food/condiment/shaker/spacespice(src)
+	new /obj/item/reagent_containers/food/condiment/pacid(src)
+	new /obj/item/reagent_containers/glass/bottle/antitoxin(src)
```

Comment:
Why not just let them ask for Medical to give them some from a vendor, or from the chemist? They don't start with soy sauce (which needs sulfuric acid) but _polytrinic acid and dylovene_ are fine? They don't start with honey which is needed for ribs, but they can have this?